cypress/e2e/inventory/actions-menu.cy.js

```diff
@@ -3,13 +3,28 @@ import InventorySearchAndFilter from '../../support/fragments/inventory/inventor
 import InventoryActions from '../../support/fragments/inventory/inventoryActions';
 import testTypes from '../../support/dictionary/testTypes';
 import devTeams from '../../support/dictionary/devTeams';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import getRandomPostfix from '../../support/utils/stringTools';
 
-describe('ui-inventory: actions', { retries: 2 }, () => {
-  beforeEach('navigates to actions', () => {
+
+const item = {
+  instanceName: `testBulkEdit_${getRandomPostfix()}`,
+  itemBarcode: getRandomPostfix(),
+};
+describe('ui-inventory: actions', () => {
+  before('login and create test data', () => {
     cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
```

Comment:
We can use function like cy.loginAsAdmin(); instead of cy.login(Cypress.env('diku...

Reply:
In this case I visit inventory page in "beforeEach" hook, because I must visit this page multiply times

Reply:
missunderstand by my side. fixed